%-------------------------------------------------------------------------------
% Preamble
%-------------------------------------------------------------------------------

\documentclass[a4paper, 12pt]{article}
\usepackage[margin=2cm]{geometry}
\usepackage[osf]{mathpazo} % palatino
%\usepackage[round]{natbib} % author-year citations
\usepackage[superscript,biblabel]{cite} % for superscript citations
\usepackage{graphicx}
\usepackage{subcaption}
\usepackage{parskip} 
\usepackage{amsmath}
\usepackage{longtable}
\usepackage{pdflscape}
\usepackage{array}
\usepackage{float}
\usepackage{url}

\pagenumbering{arabic}  
\linespread{1.3}

% figure numbering override
\renewcommand*{\thefigure}{A\arabic{figure}} % make Fig A1 not Fig 1
\renewcommand*{\thetable}{A\arabic{table}} % make Table A1 not Table 1

%-------------------------------------------------------------------------------
% Title page information
%-------------------------------------------------------------------------------

\title{Supplementary Materials from: \textit{Microchromosome fusions underpin convergent evolution of chameleon karyotypes}; journal}

\author{Marcello Mezzasalma, Jeffrey W Streicher, Fabio M. Guarino, Marc EH Jones,\\ 
Simon P Loader, Gaetano Odierna and Natalie Cooper}

\date{}

% End of preamble

\begin{document}

\maketitle

\tableofcontents

\parindent = 1.5em
\addtolength{\parskip}{.3em}

%-------------------------------------------------------------------------------
% Supplementary materials
%-------------------------------------------------------------------------------

%-----------------------------------------------------------------------------------
% figures 
%-----------------------------------------------------------------------------------
\newpage
\section{Supplementary Figures}

% figure s1
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/trees-genera-colour.png}
  \caption{Time calibrated chameleon phylogeny used in the analyses. 
  Branches for each genus have been coloured back to their most recent common ancestor.
}
  \label{fig-phylo}
\end{figure}

% figure s2
\newpage
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/trees-chromosome-properties.png}
  \caption{Distribution of chromosome properties (haploid number of chromosomes, arm number, number of macrochromosome pairs and number of microchromosome pairs) across the chameleon phylogeny. Note that the scales are different on each plot. The outlier in each case is \textit{Rieppeleon kerstenii} (n = 31). See Figure \ref{fig-phylo} for species identities.
}
  \label{fig-properties}
\end{figure} 

%-----------------------------------------------------------------------------------{}
% duplication analyses
%-----------------------------------------------------------------------------------
\newpage
\section{Chromosome evolutionary models with duplication events}

\subsection{Methods}

We used six additional models to the Constant Rates and Linear Rates models described in the main text that optimise two or more of the following parameters: (i) rate of gain of a single chromosome; (ii) rate of loss of a single chromosome; (iii) polyploidization rate; (iv) demi-polyploidization rate; (v) linear dependency between the current haploid number and the rate of gain chromosomes; and (vi) linear dependency between the current haploid number and the rate of loss chromosomes. Details of which parameters are fitted in each model are shown in Table \ref{table_A1}. All other analysis details are identical to those presented in the text.

\input{figures/model-list.tex}

\subsection{Results}

For models where duplications and demi-duplications were allowed, if \textit{Rieppeleon kerstenii} was included, the Constant Rates with duplications (and demi-duplications for estimated root frequencies) fitted best (Table \ref{table-dup-results}). If \textit{Rieppeleon kerstenii} was excluded, the best fitting model was a Constant Rates model for the fixed root frequencies, and a Constant Rates model with duplications and demi-duplications when we estimated the root frequencies (Table \ref{table-dup-results}). Overall, the best fitting model was the Constant Rates model with duplications and demi-duplications where \textit{Rieppeleon kerstenii} was excluded, and the root was estimated (AIC = 307.1; Table \ref{table-dup-results}; Figure \ref{fig-dup}).

\input{figures/table-dup-results}

% figure s3
\newpage
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/figure-chromEvol_best-model-all.png}
  \caption{Best fitting model allowing duplications, removing \textit{Rieppeleon kerstenii} and estimating the root node. Numbers at the nodes are the most frequent values obtained from 1,000 simulations. Colours represent the number of chromosomes at the tip, or the proportion of simulations with each number of chromosomes as pie charts at the nodes.
}
  \label{fig-dup}
\end{figure} 

%-----------------------------------------------------------------------------------
% ecology 
%-----------------------------------------------------------------------------------
\newpage
\section{Relationships among chromosome numbers, ecology and life history}

\subsection{Methods}

We tested for correlations between haploid numbers of chromosomes (n) and the following variables; (1) maximum snout vent length (mm; n = 86); (2) substrate (arboreal, terrestrial or multiple; n = 77); (3) reproductive mode (oviparous or viviparous; n = 71); (4) minimum clutch size (eggs; n = 58); (5) maximum clutch size (eggs; n = 58); (6) minimum breeding age (months; n = 43); (7) maximum breeding age (months; n = 43); (8) biogeographic realm (Afrotropical, Madagascar, Oriental or Palearctic; n = 86); (9) absolute latitude (decimal degrees; DD; n = 86). All ecological and life history data were taken from Meiri\cite{meiri2018traits}. Continuous variables were mean-centred and scaled to unit variance prior to analyses.

We used Bayesian phylogenetic generalised mixed models (GLMMs) with Poisson errors in the R package MCMCglmm\cite{hadfield2010mcmc} to test for correlations, including the phylogeny (as the inverse of the phylogenetic variance-covariance matrix) as a random effect to account for phylogenetic autocorrelation. We ran each MCMCglmm model for 1 x 106 iterations sampling at every 1,000 iterations and discarding the first 1 x 105 iterations as burn-in. We used the default priors for MCMCglmm ($μ = 0$ and $V = I × 1010$ for fixed effects and parameter expanded priors, and $V = 1$, $\nu = 1$, $\alpha \mu = 0$, and $\alpha V = 252$ for the phylogenetic random effects). All model parameters had a mean effective sample size (ESS; estimated using the R package coda\cite{plummer2006coda}) of over 800, and traceplots indicated that models had converged.

\subsection{Results}

We found no significant correlations among chromosome numbers and any of our ecology or life history variables (Table \ref{table-ecology}; Figures \ref{fig-ecology1} - \ref{fig-ecology2}). 

\input{figures/table-ecology}

% figure
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/ecology-genus-svl.png}
  \caption{Relationships between genus (n = 92), maximum snout vent length (SVL; n = 86), substrate (n = 83), reproductive mode (n = 77) and haploid chromosome number in chameleons. Points are coloured by genus as shown in the upper panel. The outlier at n = 31 is \textit{Rieppeleon kerstenii}.
}
  \label{fig-ecology1}
\end{figure} 

\newpage
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/mcmcglmm-figure.png}
  \caption{Results from Bayesian phylogenetic generalised linear mixed models with Poisson errors, showing posterior means of each slope estimate for the relationship between haploid chromosome number in chameleons and several explanatory variables. max\_SVL.
}
  \label{fig-ecology2}
\end{figure}



%-----------------------------------------------------------------------------------
% otus 
%-----------------------------------------------------------------------------------
\newpage
\section{Results from analyses using all OTUs in the models}

We have molecular and karyotypic data for 137 samples. Most species within the sample have a consistent karyotype, therefore our main analyses use only one representative for each species to avoid pseudoreplication, except for the five taxa which have intraspecific variation in their karyotypes. For completeness we repeated the analyses using all 137 samples.

\subsection{Chromosome evolutionary models}
When \textit{Rieppeleon kerstenii} was included the Linear Rates model fitted better than the Constant Rates model (Table \ref{table-otus}). Although rates of chromosome loss were also consistently higher than rates of chromosome gain in these models, rates of linear dependency between the current haploid number and the rate of loss of chromosomes were lower for loss than gain of chromosomes. This was regardless of root frequencies. The Constant Rates model fitted better than the Linear Rates model when \textit{Rieppeleon kerstenii} was excluded from the analyses, and rates of chromosome loss were consistently higher than rates of chromosome gain in these models (Table \ref{table-otus}). Again, this was regardless of root frequencies. Overall, the best fitting model is the Constant Rates model where \textit{Rieppeleon kerstenii} was excluded, and the root is set to 18 (Figure \ref{fig-otus-models}). 

\noindent For models where duplications and demi-duplications were allowed, if the root is estimated then the best fitting model is a Constant Rates model with both duplications and demi-duplications, regardless of whether \textit{Rieppeleon kerstenii} was excluded or not. For the other root states, the best fitting model is a Constant Rates model with duplications if \textit{Rieppeleon kerstenii} was included, and a Constant Rates model if \textit{Rieppeleon kerstenii} was excluded (Table \ref{table-otus}). 

\input{figures/table-otus}

% figure
\begin{figure}[H]
 \centering
  \includegraphics[width = \linewidth]{figures/tree-best-model-taxa.png}
  \caption{Best fitting chromosome evolution model using taxa as tips, removing \textit{Rieppeleon kerstenii} and using n = 18 as the root node. Numbers at the nodes are the most frequent values obtained from 1,000 simulations. Colours represent the number of chromosomes at the tip, or the proportion of simulations with each number of chromosomes as pie charts at the nodes.
}
  \label{fig-otus-models}
\end{figure} 

\subsection{Fissions and fusions and ITS}
There was a significant negative correlation between the haploid number of chromosomes and the number of macrochromosome pairs (GLM: $\chi^2$ = 12.49, df = 1,134, p $<$ 0.001; Figure \ref{fig-otus-macro}A), and a significant positive correlation between the haploid number of chromosomes and the number of microchromosome pairs (GLM: $\chi^2$ = 72.89, df = 1,134, p $<$ 0.001; Figure \ref{fig-otus-macro}B). Additionally, there was a significant relationship between the number of micro- and macro- chromosome pairs (GLM: $\chi^2$ = 169.57, df = 1,134, p $<$ 0.001; Figure \ref{fig-otus-macro}C). There was a significant negative correlation between ITS and the haploid number of chromosomes (GLM: F = 59.91, df = 1,72, p $<$ 0.001; Figure \ref{fig-otus-its}A), a significant positive correlation between ITS and the number of macrochromosome pairs (GLM: F = 16.30, df = 1,72, p $<$ 0.001; Figure \ref{fig-otus-its}B), and a significant relationship between ITS and the number of microchromosome pairs (GLM: F = 61.87, df = 1,72, p $<$ 0.001; Figure \ref{fig-otus-its}C). 

% figure
\begin{figure}[H]
 \centering
  \includegraphics[width = \linewidth]{figures/micro-macro-chromosomes.png}
  \caption{Correlations among the haploid number of chromosomes (n), numbers of macrochromosome pairs and numbers of microchromosome pairs in chameleons. Fitted lines and standard errors are the outputs from generalised linear models with Poisson errors. The outlier at n = 31 is \textit{Rieppeleon kerstenii}.
}
  \label{fig-otus-macro}
\end{figure} 


% figure
\newpage
\begin{figure}[H]
 \centering
  \includegraphics[width = \linewidth]{figures/ITS-chromosomes.png}
  \caption{Correlations among ITS and the haploid number of chromosomes (n), numbers of macrochromosome pairs and numbers of microchromosome pairs in chameleons. Fitted lines and standard errors are the outputs from generalised linear models with quasipoisson errors.
}
  \label{fig-otus-its}
\end{figure} 


\subsection{Phylogenetic patterns}
Differences in chromosome number did not increase with phylogenetic distance (Figure \ref{fig-otus-pairwise}); even some of the most distantly related taxa in our phylogeny shared the same chromosome numbers. Large differences in chromosome numbers, however, only occur at moderate to large phylogenetic distances. Simulations give a reasonable approximation of observed chromosome numbers at the tips of the phylogeny, however, model predictions do not account well for the large numbers of species with n = 11, n = 12 and n = 18 chromosomes (Figure \ref{fig-otus-predicted}). 

% figure
\newpage
\begin{figure}[H]
 \centering
  \includegraphics[width = \linewidth]{figures/taxon-pairs-distances.png}
  \caption{Phylogenetic distance (in millions of years) in relation to differences in chromosome numbers (2n) for each pair of taxa in the chameleon tree. Note that the cluster of values with chromosome differences greater than 10 are comparisons of various taxa with \textit{Rieppeleon kerstenii} (2n = 62).
}
  \label{fig-otus-pairwise}
\end{figure} 

% figure
\newpage
\begin{figure}[H]
 \centering
  \includegraphics[width = \linewidth]{figures/chromevol-simulations-numbers.png}
  \caption{Distribution of observed (grey) and predicted (blue) haploid numbers of chromosomes in chameleons, excluding \textit{Rieppeleon kerstenii}. Predicted values are based on 1,000 simulations using the optimised parameters taken from the best fitting model identified in the chromosome evolution analyses above (Constant Rates, removing \textit{Rieppeleon kerstenii}, and using n = 18 as the root node). Observed values were multiplied by 1,000 to aid comparisons.
}
  \label{fig-otus-predicted}
\end{figure} 

%-------------------------------------------------------------------------------
\newpage
\section{Calibration points for phylogeny}
%-------------------------------------------------------------------------------

Please note that our constraints differ from those used by Tolley \textit{et al.}\cite{tolley2013large}. Given that (accurately identified) fossils (with apomorphies) provide hard minima for the emergence of clades we consider conservative constraints to be constraints with shallower dates.
 
\subsection{External points}
 
\subsubsection*{Node 1. (Lepidosauria) – 250 Ma}
For Lepidosauria we used a point fix of 250 Ma. Note that this node by definition also equates to both total group Squamata and total group Rhynchocephalia\cite{evans2003feet}. The oldest known fossil material representing total-group Squamata is currently considered to be \textit{Megachirella wachtleri} from the Anisian of Italy\cite{renesto2003new,simoes2018origin}. If this taxon is indeed a stem squamate it provides a hard minimum of 242 Ma for total group Squamata and thus also Lepidosauria. The oldest known fossil material representing total group Rhynchocephalia is not much younger and is represented by partial skull and jaw bones from the Ladinian of Germany\cite{jones2013integration,schoch2018tetrapod}. This material has not been fully described but probably includes at least three different genera which illustrated that Rhynchocephalia must have diversified by the Middle Triassic. Fossils representing stem lepidosaurs are known from both the Early and Middle Triassic\cite{evans2009small,schoch2018tetrapod}. Previous molecular divergence estimates for Lepidosauria have differed greatly with median estimates as deep as the Early Permian (e.g. 281 Ma) and as shallow as the Late Triassic (e.g. 227 Ma\cite{jones2013integration}) and associated confidence intervals can exceed 20 million years. Estimates shallower than the Middle Triassic can be rejected with empirical evidence from the fossil record\cite{jones2013integration,simoes2018origin}. Much older origin times deep within the Permian are possible (e.g.\cite{simoes2018origin}) but there are currently no convincing crown-group lepidosaurs known from this time period.  

\subsubsection*{Node S (Squamata) – not constrained}
We did not constrain the node for crown group Squamata. For the origin and divergence of crown group Squamata, molecular analyses have suggested a wide range of possible dates spanning the Permian, Triassic, and Jurassic (see\cite{jones2013integration}). However, most studies suggest a median date within 15 Ma of the Triassic-Jurassic boundary\cite{jones2013integration,simoes2018origin,burbrink2020interrogating}. The early history of crown-group lizards is poorly known but fossil material is known from the Middle Jurassic (e.g.\cite{evans1998crown,evans2003feet,evans2008skull}). 

\subsubsection*{Node U. (Unidentata) – not constrained}
We did not constrain the node for crown group Unidentata.
 
\subsubsection*{Node 2. (Xantusia-Cordylus) – 62.5 Ma (min) 100 Ma (max)}
We constrain the divergence between cordylids and xantusiids to 61 Ma based on \textit{Palaeoxantusia fera} from the Palaeocene rocks of the Gidley and Silberling quarries of USA\cite{smith2009eocene}. The material exhibits several traits suggestive of a close affinity to xantusiids\cite{smith2009eocene,smith2011long}. The Gidley and Silberling rocks are suspected to be correlated with magnetic polarity chron C27r\cite{butler1987magnetic} now inferred to be 63.5 and 62.5 Mya\cite{gradstein2012geologic}. Therefore, we infer a hard minimum of 62.5 Ma.

\subsubsection*{Node 3. (Laterata) – 138.7 Ma (min) 140.1 (max)}
The node defining Laterata (Teiioidea, Amphisbaenia, and Lacertidae) was constrained to 138.7 Ma. The oldest known fossil material that can be referred to Lacertidae or Teiiodea is \textit{Purbicella ragei} from the Cherty Freshwater Beds of the Lulworth Formation of the Purbeck Group, England UK\cite{evans2012new}. Key morphological characters present in the material include the presence of a pterygoid lappet on the quadrate, a narrow pyriform recess, a jugal located dorsal to the maxilla, and a parietal with anterior tabs that slot into the underside of the frontal\cite{conrad2008phylogeny,evans2003feet}. The Cherty Freshwater Beds are part of the Purbeck Group and considered to be Berriasian in age, Lower Cretaceous\cite{radley2012wealden}. The upper boundary of the Berriasian is considered to be 139.4 $\pm$ 0.7 Ma\cite{gradstein2012geologic}therefore, providing a hard minimum of 138.7. This constraint is older than the date of 122 $\pm$ 1.0 Ma based on \textit{Ptilodon}\cite{nydam2002lizards,winkler1990early} used in Tolley \textit{et al.}\cite{tolley2013large}.
  
\subsubsection*{Node 4. (Lacertibaenia) – 61 Ma (min) 100 (max)}
We constrain the node representing the divergence of Lacertibaenia (Amphisbaenia and Lacertidae) to 61 Ma based on \textit{Plesiorhineura tsentasi} from the Palaeocene of USA\cite{sullivan1985new}. Older fossil material named \textit{Hodzhakulia} (112 $\pm$ 1.0 Ma) used in Tolley \textit{et al.}\cite{tolley2013large} is less certainly amphisbaenian and may therefore be a problematic calibration point\cite{hipsley2009integration}.
 
\subsubsection*{Node 5. (Anguimorpha) – 148 Ma (min) 155 Ma (max)}
Anguimorpha 148 Ma, minimum hard bound based on \textit{Dorsetisaurus} sp. from the Late Jurassic Morrison Formation of North America\cite{prothero1980late,evans1999upper,conrad2008phylogeny,jones2013integration}. Combined evidence analysis places \textit{Dorsetisaurus purbeckensis} as a non-anguiform anguimorph and therefore the oldest known representative of Anguimorpha\cite{conrad2011combined}.
 
\subsubsection*{Node 6. (crown Serpentes) – 93.9 Ma (min) 100.5 Ma (max)}
The divergence between \textit{Liotyphlops} and \textit{Dinodon} (crown Serpentes) was constrained to a hard minimum of 93.9 Ma and maximum of 100.5 Ma based on \textit{Haasiophis terrasanctus} from the Late Cretaceous of the Middle East\cite{tchernov2000fossil,rieppel2003anatomy}. This date is the same as used elsewhere (e.g. 93.9 Ma used by\cite{head2015fossil,hsiang2015origin})
 
\subsubsection*{Node I. (Iguania) – not constrained}
The oldest stem iguanians are uncertain\cite{bolet2013fossil}. Fossils from the Early Jurassic previously referred to iguanians may belong to another clade (e.g.\cite{jones2013integration}).
 
\subsubsection*{Node 7. (Anguioidea) – 74.5 Ma (min) 76.6 (max)}
The node representing the most recent common ancestor (MRCA) of Xenosauridae and Anguidae was constrained to a minimum of 74.5 Ma. The oldest known fossil material representing Anguidae is \textit{Odaxosaurus}\cite{nydam2013lizards} from the Kaiparowits Formation, locality OMNH V5, of Garfield County, Utah, USA. An integrated analysis using morphological and molecular data recovers \textit{Odaxosaurus piger} as the sister taxon to Glyptosaurinae\cite{conrad2011combined}. Extensive stratigraphic and isotope analyses of the Kaiparowits Formation indicate an age range of 74.5–76.6 Myr\cite{roberts200540ar,roberts2013kaiparowits}. Therefore providing a hard minimum estimate of 74.5. 
 
\subsubsection*{Node 8. (Pleurodonta) – 70 Ma (min) 71.2 (max)}
Crown-group pleurodont Iguania is constrained to a minimum date of 70 Ma. The earliest certain representative of crown-group pleurodont iguanians is \textit{Saichangurvel}\cite{conrad2007complete} from the Late Campanian (72.5 $\pm$ 2.5 Ma) of Mongolia. This constraint was also used by Tolley \textit{et al.}\cite{tolley2013large} with a date of 70.6 $\pm$ 0.6 Mya.
 
\subsection{Internal points}
\subsubsection*{Node C. (stem Chamaeleonidae) – not constrained}
We do not constrain this node. Although a fossil skeleton (specimen JZC Bu154) from the Cretaceous of Myanmar (99 Ma) has been referred to the stem of Chamaeleonidae\cite{daza2016mid}, and previously used to constrain analyses (e.g.\cite{skawinski2017evolution}), this material is currently in the process of a more detailed study and is almost certainly not even a lepidosaur (Susan Evans \textit{pers comm.} 2022).
 
\subsubsection*{Node 9. (\textit{Calumma}) – 16 Ma (min) 20 Ma (max)}
The genus \textit{Calumma} is constrained to a hard minimum date of 16 Ma. The earliest known representative of this genus is a nearly complete articulated skull, \textit{Calumma benovskyi}, from the Early Miocene Hiwegi Formation of Rusinga Island Kenya\cite{vcervnansky2020only}. Detailed morphological comparisons and phylogenetic analyses confidently place this specimen within \textit{Calumma}\cite{vcervnansky2020only}.
 
\subsubsection*{Node 10. (\textit{Chamaeleo}) – 16.6 Ma (min) 20 Ma (max)}
The genus \textit{Chamaeleo} is constrained to a minimum date of 16.6 Ma. The earliest known representatives of this genus come from units referred to the 3 to 6 mammal Neogene zones (MN)\cite{vcervnansky2010revision,bolet2013fossil}. Zone 3 corresponds to approximately 16.6 to 20 Ma and therefore provides a hard minimum date of 16.6.\cite{van2011biostratigraphy}. The origin maybe older but we prefer to use the shallower data so that the error is more certainly unidirectional.

\newpage
\section{Supplementary References}
 
\bibliography{chameleons}
\bibliographystyle{bibstyle}

\end{document}


%-------------------------------------------------------------------------------\
% karyotyping
%-------------------------------------------------------------------------------

\section{Chromosome analyses}
\subsection{Cytogenetic analysis and selection of chromosome markers}
Chromosome were obtained using the traditional air-drying method (see e.g. Sidhom et al. 2020) on cell suspensions preserved in Carnoy’s buffer (methyl alcohol – glacial acetic acid 3:1), then the samples were studied with a combination of standard colorations and molecular cytogenetics as described as follows. Karyotypes were reconstructed using the standard Giemsa coloration (5\% at pH 7). NOR loci were detected using Ag-NOR staining following Howell & Black (1980) and Fluorescence in Situ Hybridization (FISH) of telomeric (TTAGGG)n repeats as reported in Sidhom et al. (2020). In brief, DNA probes of the telomeric repeats were produced and labelled using biotin 11-dUTP via PCR according to the protocol reported by Ijdo et al. (1991). Detection of the FISH signals was performed using Streptavidin, Alexa Fluor™ 488 conjugate (ThermoFisher), and counter-stained with DAPI. Metaphase plates were identified and recorded with an epifluorescent microscope (Axioscope Zeiss) equipped with a digital analysis system. 

Chromosome characters used in our inferences with phylogenetic comparative methods (see below) were: haploid chromosome number, arm number, macro- and microchromosome number, position of NOR loci, number of ITS and the morphology of each macrochromosome pair following the traditional classification based on the centromeric index (metacentric, submetacentric, subtelocentric and telocentric), following Levan et al. (1964).
The morphology of heteromorphic sex chromosome pairs (only known in Furcifer) was classified according to the morphology of the homomorphic Z elements, while the morphology of putative heteromorphic autosome pairs (only in Calumma) was classified according to the element with higher centromeric index (CI).

Count of ITS was performed considering only interstitial and peritelomeric signals present in both homologs, which are considered true ITS; Bolzán 2011; Chirino et al. 2017), thus excluding centromeric and standard telomeric positive signals.

To produce a comprehensive assessment of chromosomal evolutionary dynamics in chameleons our newly generated chromosome data were added to those available from literature. Our complete chromosome dataset included karyotypes of 95 OTUs, covering more than 50\% of those included in the phylogenetic analysis. A complete list of the values of all the chromosome characters used in phylogenetic inference with comparative methods and relative references to literature data included are reported in Supplementary table 2.


Chromosome number was found to be constant (2n = 36, AN = 48) in all the Brookesia species (n = 13) and in Palleon nasus, and invariably with 6 macro- and 12 microchromosome pairs. All the macrochromosome pairs resulted metacentric with the only exception of the 2nd submetacentric pair in P. nasus. No heteromorphic chromosome pairs were found in any Brookesia species or in P. nasus. No ITS signals were found in the karyotype of any Brookesia species or in P. nasus.

In the genus Calumma, karyotypes varied from 2n = 38 (in C. tarzan) to 2n = 28 (in C. nasutum, C. boettgeri and C. fallax) (AN = 40-56), with 2n = 36 resulting the most common chromosome configuration in the genus (in 10 out of 24 species studied). Karyotype of 2n = 34, 32 and 30 chromosomes were showed by four, three and four species, respectively. A variable number of macro- (6-9) and microchromosome pairs (7-13) was also recorded among the studied species of the genus. Chromosome morphology is also variable in Calumma, macrochromosome pairs are generally biarmed (from meta- to subtelocentric pairs), but telocentric elements were observed in Calumma sp. 1 (7th pair). A heteromorphic pair was observed in the karyotype of C. crypticum. NOR bearing pairs were detected in 14 species, on the 1st (3 species) or 2nd chromosome pair (11 species). ITS signals (n = 2-4) were found in six species showing karyotypes from 2n = 36 to 2n = 28.

The genus Furcifer showed karyotypes ranging from 2n = 34 (only in F. balteatus) to 2n = 22 (the most common karyotype configuration in the genus, found in eight out of 16 species studied) (AN = 34-54). Karyotypes of 2n = 28 were showed by two species (F. wilsii and F. labordi), 2n = 26 by one species (F. campani) and 2n = 24 by five species. Macrochromosome pairs ranged from six to 13 pairs, while microchromosomes from one to 11 pairs. Chromosome morphology is mostly biarmed in the genus (meta- and submetacentric pairs are common in the genus), but 1-2 telocentric pairs are present in the karyotype of F. belalandensis and F. minor. Heteromorphic chromosome pairs were found in seven species, usually on the last macrochromosome pair. In the species here analysed, loci of NOR are always on macrochromosomes (1st, 2nd, 3rd or 4th pair). ITS signals (n = 2-14) were found in 11 species with 2n = 34-22.

