%-------------------------------------------------------------------------------
%DIF LATEXDIFF DIFFERENCE FILE
%DIF DEL chameleons_evol.tex      Wed Mar  8 13:56:17 2023
%DIF ADD chameleons_evol_R1.tex   Tue Apr 11 13:21:49 2023
% Preamble
%-------------------------------------------------------------------------------

\documentclass[a4paper, 12pt]{article}
\usepackage{ms}            % load the template
\usepackage[osf]{mathpazo} % palatino
\usepackage[round]{natbib} % author-year citations
\usepackage[superscript,biblabel,nomove]{cite} % for superscript citations
\usepackage{graphicx}
\usepackage{subcaption}
\usepackage{parskip} 
\usepackage{amsmath}
\usepackage{longtable}
\usepackage{pdflscape}

%DIF 17a17
 %DIF > 
%DIF -------
\pagenumbering{arabic}  
\linespread{1.66}

%-------------------------------------------------------------------------------
% Begin document
%-------------------------------------------------------------------------------
%DIF PREAMBLE EXTENSION ADDED BY LATEXDIFF
%DIF UNDERLINE PREAMBLE %DIF PREAMBLE
\RequirePackage[normalem]{ulem} %DIF PREAMBLE
\RequirePackage{color}\definecolor{RED}{rgb}{1,0,0}\definecolor{BLUE}{rgb}{0,0,1} %DIF PREAMBLE
\providecommand{\DIFadd}[1]{{\protect\color{blue}\uwave{#1}}} %DIF PREAMBLE
\providecommand{\DIFdel}[1]{{\protect\color{red}\sout{#1}}}                      %DIF PREAMBLE
%DIF SAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddbegin}{} %DIF PREAMBLE
\providecommand{\DIFaddend}{} %DIF PREAMBLE
\providecommand{\DIFdelbegin}{} %DIF PREAMBLE
\providecommand{\DIFdelend}{} %DIF PREAMBLE
%DIF FLOATSAFE PREAMBLE %DIF PREAMBLE
\providecommand{\DIFaddFL}[1]{\DIFadd{#1}} %DIF PREAMBLE
\providecommand{\DIFdelFL}[1]{\DIFdel{#1}} %DIF PREAMBLE
\providecommand{\DIFaddbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFaddendFL}{} %DIF PREAMBLE
\providecommand{\DIFdelbeginFL}{} %DIF PREAMBLE
\providecommand{\DIFdelendFL}{} %DIF PREAMBLE
\newcommand{\DIFscaledelfig}{0.5}
%DIF HIGHLIGHTGRAPHICS PREAMBLE %DIF PREAMBLE
\RequirePackage{settobox} %DIF PREAMBLE
\RequirePackage{letltxmacro} %DIF PREAMBLE
\newsavebox{\DIFdelgraphicsbox} %DIF PREAMBLE
\newlength{\DIFdelgraphicswidth} %DIF PREAMBLE
\newlength{\DIFdelgraphicsheight} %DIF PREAMBLE
% store original definition of \includegraphics %DIF PREAMBLE
\LetLtxMacro{\DIFOincludegraphics}{\includegraphics} %DIF PREAMBLE
\newcommand{\DIFaddincludegraphics}[2][]{{\color{blue}\fbox{\DIFOincludegraphics[#1]{#2}}}} %DIF PREAMBLE
\newcommand{\DIFdelincludegraphics}[2][]{% %DIF PREAMBLE
\sbox{\DIFdelgraphicsbox}{\DIFOincludegraphics[#1]{#2}}% %DIF PREAMBLE
\settoboxwidth{\DIFdelgraphicswidth}{\DIFdelgraphicsbox} %DIF PREAMBLE
\settoboxtotalheight{\DIFdelgraphicsheight}{\DIFdelgraphicsbox} %DIF PREAMBLE
\scalebox{\DIFscaledelfig}{% %DIF PREAMBLE
\parbox[b]{\DIFdelgraphicswidth}{\usebox{\DIFdelgraphicsbox}\\[-\baselineskip] \rule{\DIFdelgraphicswidth}{0em}}\llap{\resizebox{\DIFdelgraphicswidth}{\DIFdelgraphicsheight}{% %DIF PREAMBLE
\setlength{\unitlength}{\DIFdelgraphicswidth}% %DIF PREAMBLE
\begin{picture}(1,1)% %DIF PREAMBLE
\thicklines\linethickness{2pt} %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\framebox(1,1){}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,0){\line( 1,1){1}}}% %DIF PREAMBLE
{\color[rgb]{1,0,0}\put(0,1){\line(1,-1){1}}}% %DIF PREAMBLE
\end{picture}% %DIF PREAMBLE
}\hspace*{3pt}}} %DIF PREAMBLE
} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbegin}{\DIFaddbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddend}{\DIFaddend} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbegin}{\DIFdelbegin} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelend}{\DIFdelend} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbegin}{\DIFOaddbegin \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbegin}{\DIFOdelbegin \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelend}{\DIFOaddend \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddbeginFL}{\DIFaddbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOaddendFL}{\DIFaddendFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelbeginFL}{\DIFdelbeginFL} %DIF PREAMBLE
\LetLtxMacro{\DIFOdelendFL}{\DIFdelendFL} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddbeginFL}{\DIFOaddbeginFL \let\includegraphics\DIFaddincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFaddendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelbeginFL}{\DIFOdelbeginFL \let\includegraphics\DIFdelincludegraphics} %DIF PREAMBLE
\DeclareRobustCommand{\DIFdelendFL}{\DIFOaddendFL \let\includegraphics\DIFOincludegraphics} %DIF PREAMBLE
%DIF END PREAMBLE EXTENSION ADDED BY LATEXDIFF

\begin{document}
\modulolinenumbers[1]   % Line numbering on every line

%DIF < \mstitlepage
%DIF > \mstitlepage===

\parindent = 1.5em
\addtolength{\parskip}{.9em}

\raggedright

%-------------------------------------------------------------------------------
% Title
%------------------------------------------------------------------------------

\section{Microchromosome fusions underpin convergent evolution of chameleon karyotypes}

%-------------------------------------------------------------------------------
% Abstract
%-------------------------------------------------------------------------------
\DIFdelbegin %DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend \DIFaddbegin \newpage
\DIFaddend \section{Abstract}

Evolutionary shifts in chromosome compositions (karyotypes) are major drivers of lineage and genomic diversification. 
Fusion of ancestral chromosomes is one \DIFdelbegin \DIFdel{hypothesised }\DIFdelend \DIFaddbegin \DIFadd{hypothesized }\DIFaddend mechanism for the evolutionary reduction of the total chromosome number; a frequently implied karyotypic shift. 
\DIFdelbegin \DIFdel{However, empirical }\DIFdelend \DIFaddbegin \DIFadd{Empirical }\DIFaddend tests of this hypothesis \DIFdelbegin \DIFdel{are rare, as they }\DIFdelend require model systems with variable karyotypes, known chromosome features, and a robust phylogeny. 
Here we used chameleons, diverse lizards with exceptionally variable karyotypes (2n = 20-62), to test whether chromosomal fusions explain the repeated evolution of \DIFdelbegin \DIFdel{smaller-than-ancestral karyotypes }\DIFdelend \DIFaddbegin \DIFadd{karyotypes with fewer chromosomes than ancestral karyotypes}\DIFaddend . 
Using a multidisciplinary approach including cytogenetic analyses and phylogenetic comparative methods, we found that a model of constant loss through time best explained chromosome evolution across the chameleon phylogeny. 
Next we tested whether fusions of microchromosomes into macrochromosomes explained these evolutionary losses using \DIFdelbegin \DIFdel{generalised }\DIFdelend \DIFaddbegin \DIFadd{generalized }\DIFaddend linear models. 
Multiple comparisons supported microchromosome fusions as the predominant agent of evolutionary loss. 
We further compared our results to various natural history traits and found no correlations. 
As such, we infer that the tendency of microchromosomes to fuse was a quality of the ancestral chameleon genome, and that the genomic predisposition of ancestors is a more substantive predictor of chromosome change than the ecological, physiological, and biogeographical factors involved in their diversification.

\textbf{Keywords: chromosome evolution, cytogenetics, molecular phylogenetics, ancestral state estimation}

%-------------------------------------------------------------------------------
% Main text
%-------------------------------------------------------------------------------

\section{Introduction}\label{main}

Chromosomal evolution is a major driver of biodiversity with inter- and intra-chromosome rearrangements contributing to speciation and lineage diversification \citep{pellestor2020chromoanagenesis}. 
Vertebrate animals have experienced variable patterns of chromosome recombination and change, leading to some animal groups with conserved, evolutionarily static karyotypes and others with diverse, evolutionarily plastic karyotypes (e.g. \citealt{graphodatsky2011genome,neto2011extensive,mezzasalma2019changes,degrandi2020distribution,mayrose2021evolution}).
Why do some animals have conserved karyotypes \DIFdelbegin \DIFdel{whilst others }\DIFdelend \DIFaddbegin \DIFadd{whereas others have }\DIFaddend divergent chromosome arrangements? 
The answer is likely complex, but is partially explained by different evolutionary tendencies toward either the maintenance of a particular genomic structure or the acquisition of new chromosomal and genetic characters via chromosome rearrangements (chromosome fissions, fusions and inversions; \citealt{crombach2007chromosome,amorim2021new}).

The evolutionary dynamics of chromosome rearrangements have been studied for over a century in multiple vertebrate groups \citep{damas2021vertebrate}. 
Squamate reptiles (lizards and snakes) have been a focus of evolutionary cytogenetics as they have exceptionally diverse karyotypes that can be either ‘asymmetrical’ or ‘symmetrical’ based on the occurrence of chromosomes of distinct dimensional classes \citep{stebbins1950chapter,white1973}. 
Asymmetrical karyotypes contain chromosomes that can be binned into categories of either large macrochromosomes or small, dot-shaped microchromosomes.
In contrast, symmetrical karyotypes contain chromosomes that cannot be binned into size classes as their lengths vary near continuously. 
Evolutionary transitions between asymmetrical and symmetrical chromosome arrangements are \DIFdelbegin \DIFdel{hypothesised }\DIFdelend \DIFaddbegin \DIFadd{hypothesized }\DIFaddend to have occurred in both directions, primarily by the mechanisms of fission and fusion (e.g. \citealt{olmo2008trends,srikulnath2015karyotype}). 
While most major squamate groups have only asymmetrical or only symmetrical karyotypes (i.e. one or the other), a handful of squamates exhibit both \citep{mezzasalma2021lizards}, making them valuable models for understanding the evolutionary mechanisms that allow for (or restrict) evolutionary changes in karyotype.    

True chameleons (Squamata: \DIFaddbegin \DIFadd{Iguania: }\DIFaddend Chamaeleonidae) are karyologically diverse with substantial variability in chromosome number (2n = 20-62), morphology, macro:micro-chromosome ratio, localisation of different chromosome markers and evidence of evolutionary transitions between asymmetrical and symmetrical karyotype structures \citep{rovatsos2017evolution,nielsen2018dynamic}.
Previous research has speculated that the variability of chameleon karyotypes is explained by a tendency for ancestral chromosome disappearance via microchromosome fusion \citep{rovatsos2017evolution}, causing the repeated evolution of symmetrical karyotypes from ancestors with asymmetrical arrangements. 
However, this hypothesis has not been tested within an empirical, phylogenetic framework or using mechanistic expectations of chromosome loss over evolutionary time. 
Furthermore, changes in the chromosome number lacked evidence from interstitial telomeric sequences (ITSs), whose presence/absence was not considered phylogenetically informative or able to reliably describe past chromosome rearrangements \citep{rovatsos2017evolution}.

Here, we employed a multidisciplinary approach to analyse the evolutionary pathways of chromosome changes in the family Chamaeleonidae, including a combination of cytogenetic analyses and phylogenetic comparative methods. 
Specifically, we tested whether evolutionary gain or loss of chromosomes via chromosome fission or fusion of microchromosomes into macrochromosomes, best explains karyotype evolution in chameleons. 
We \DIFdelbegin \DIFdel{hypothesised }\DIFdelend \DIFaddbegin \DIFadd{hypothesized }\DIFaddend that if chromosome loss via microchromosome fusion has been the most common mode of change in chameleon karyotypes we should observe two patterns. 
First, we expected that microchromosome number would be positively correlated with the overall number of chromosomes and that macrochromosome number would be negatively correlated with the overall number of chromosomes. 
We also expected the number of microchromosomes to be negatively correlated with the number of macrochromosomes. 
Second, given that ITSs are typically remnants of past fusion events \citep{bolzan2017interstitial}, we expected that the number of ITSs would be positively correlated with the number of macrochromosomes. 
Similarly, the number of ITSs should be negatively correlated with the overall number of microchromosomes and chromosomes (in general), if microchromosome fusion is common. 

To place our understanding of chromosome evolution in an organismal context, we compared evolutionary and karyotypic divergence directly. 
We also used phylogenetic \DIFdelbegin \DIFdel{generalised }\DIFdelend \DIFaddbegin \DIFadd{generalized }\DIFaddend linear mixed models to test for associations with multiple life history traits of chameleons (latitude, geographic realm, maximum breeding age, minimum breeding age, maximum clutch size, minimum clutch size, reproductive mode, substrate preference, and body size). 
Collectively, an integrative interpretation of our analyses, including multi-locus molecular phylogenetic analysis, both conventional and molecular cytogenetics and phylogenetic comparative methods, supports the idea that a common evolutionary mechanism led to the repeated and independent evolution of karyotypes with a reduced total chromosome number in extant chameleons.

%-------------------------------------------------------------------------------
\section{Materials and Methods}

We combined novel data from 57 Malagasy chameleon species (32 of which have not previously been karyotyped) with existing molecular and karyotypic data for chameleons ([details removed to anonymize submission]). 
The dataset contained 83 described species of Chamaeleonidae, three undescribed \textit{Calumma} species, and one outgroup taxon (\textit{Leiolepis belliana}). 
Therefore, our sample represents a significant proportion of the $\sim$200 extant chameleon species \citep{uetz2022}. 
Some species have multiple samples, meaning we have data for 137 operational taxonomic units (OTUs). 
The majority of these OTUs have conserved karyotypes within species, so we randomly selected one OTU for each species to avoid pseudoreplication.
\textit{Calumma brevicorne}, \textit{Calumma fallax}, \textit{Calumma parsonii}, and \textit{Furcifer verrucosus} are each represented by two samples with unique karyotypes, and \textit{Calumma gallus} is represented by three samples with unique karyotypes, therefore, we included these OTUs as additional taxa, thus we have 93 taxa in our main analyses. 
We repeated all analyses below using the full set of 137 OTUs (see Supplementary Materials: Table \DIFdelbegin \DIFdel{A4, Figures A6 }\DIFdelend \DIFaddbegin \DIFadd{A1, Figures A1 }\DIFaddend - \DIFdelbegin \DIFdel{A10}\DIFdelend \DIFaddbegin \DIFadd{A4}\DIFaddend ).

\subsection{Phylogenetic inference}
For our novel data, we used PCR to amplify the nuclear RAG1, CMOS and PRLR, and the mitochondrial 16S, ND2 and ND4 markers used in Tolley \DIFdelbegin \textit{\DIFdel{et al.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \DIFadd{et al. }\DIFaddend (\citeyear{tolley2013large}), and then combined these with homologous sequences from GenBank. 
All newly generated sequences were deposited in GenBank (accession numbers will be provided after manuscript acceptance). 
Our final alignment included 5,488 nucleotide positions and 303 sequences including outgroup taxa. 
We inferred a \DIFdelbegin \DIFdel{Maximum Likelihood (ML) phylogeny using RAxML 8.2.12 \mbox{%DIFAUXCMD
\citep{stamatakis2014raxml} }%DIFAUXCMD
under the GTRCAT model with }\DIFdelend \DIFaddbegin \DIFadd{Bayesian phylogeny using MrBayes 3.2.7 \mbox{%DIFAUXCMD
\citep{ronquist2012mrbayes}}%DIFAUXCMD
.
To identify the most appropriate model partitioning scheme for our alignment, we used PartionFinder 2.1.1 \mbox{%DIFAUXCMD
\citep{lanfear2017partitionfinder}}%DIFAUXCMD
. 
We ran the analysis for 10 million generations sampling every }\DIFaddend 1,000 \DIFdelbegin \DIFdel{bootstrap replicates}\DIFdelend \DIFaddbegin \DIFadd{generations to avoid autocorrelation}\DIFaddend . 
We constrained the monophyly of \textit{Calumma} $+$ \textit{Furcifer} (as in \citealt{tolley2013large}), and based on genus-level phylogenetic relationships identified with a wider selection of molecular markers. 
We also performed unconstrained phylogenetic analyses for comparative purposes.
To \DIFaddbegin \DIFadd{diagnose convergence and mixing in our Bayesian analyses we used RWTY \mbox{%DIFAUXCMD
\citep{warren2017rwty} }%DIFAUXCMD
in R \mbox{%DIFAUXCMD
\citep{R}}%DIFAUXCMD
. 
After removing 50 trees from each chain as burnin, we found that the approximate effective sample sizes (ESS) for both chains were $>$380 (constrained analysis) and $>$290 (unconstrained analysis). 
Split frequency comparisons revealed that the agreement of the two chains was high with significant correlations detected in the constrained (r = 0.98, ASDSF = 0.0097) and unconstrained (r = 0.99, ASDSF = 0.0057) analyses. 
 }

\DIFadd{Next, to }\DIFaddend date our constrained phylogenies, we used the \DIFdelbegin \DIFdel{penalised }\DIFdelend \DIFaddbegin \DIFadd{penalized }\DIFaddend likelihood method implemented in treePL \citep{smith2012treepl}, including 10 calibration points (eight external and two internal) based on available literature (see Supplementary Materials for details), as minimum hard bounds: (1) Lepidosauria - 250 Ma; (2) \textit{Xantusia-Cordylus} - 62.5 Ma; (3) Laterata - 138.7 Ma; (4) Lacertibaenia - 61 Ma; (5) Anguimorpha - 148 Ma; (6) crown Serpentes - 93.9 Ma; (7) Anguioidea - 74.5 Ma; (8) Pleurodonta -70 Ma; (9) \textit{Calumma} - 16 Ma and (10) \textit{Chamaeleo} - 16.6 Ma. 
\DIFdelbegin \DIFdel{Before analysis, we pruned the }\DIFdelend \DIFaddbegin \DIFadd{We ran two optimization phases in treePL \mbox{%DIFAUXCMD
\citep{smith2012treepl} }%DIFAUXCMD
using the consensus topology (with branch lengths) from the MrBayes 3.2.7 analysis. 
The first `priming' step allowed us to determine the optimal search settings (opt = 2, moredetail, optad = 4, moredetailed) which we then implemented in a subsequent `thorough'cross validation analysis that selected an appropriate smoothing parameter. 
A thorough analysis in treePL ensures that the analysis runs until convergence of log-likelihoods is observed. 
We used the optimal smoothing parameter (smooth = 10) to conduct a final thorough analysis which was used as the final time tree.
Before subsequent analyses, we pruned the time-calibrated }\DIFaddend phylogeny so it contained only the 96 taxa in our karyotype dataset.

\subsection{Cytogenetic analysis}
Chromosomes for our samples were obtained using the \DIFdelbegin \DIFdel{traditional }\DIFdelend \DIFaddbegin \DIFadd{conventional }\DIFaddend air-drying method (e.g. \citealt{sidhom2020karyological}) on cell suspensions preserved in Carnoy’s buffer (methyl alcohol – glacial acetic acid 3:1). 
One of us (MM) then reconstructed karyotypes using standard Giemsa coloration (5\% at pH 7). 
The chromosome \DIFdelbegin \DIFdel{characters }\DIFdelend \DIFaddbegin \DIFadd{traits }\DIFaddend we recorded were (i) haploid chromosome number, (ii) arm number, (iii) macro- and (iv) microchromosome number, (v) position of NOR loci (detected using Ag-NOR staining following Howell and Black \DIFdelbegin \DIFdel{; \mbox{%DIFAUXCMD
\citealt{howell1980controlled} }%DIFAUXCMD
and Fluorescence in Situ }\DIFdelend \DIFaddbegin \DIFadd{(\mbox{%DIFAUXCMD
\citeyear{howell1980controlled}}%DIFAUXCMD
) and Fluorescence }\textit{\DIFadd{in situ}} \DIFaddend Hybridization (FISH) of telomeric (TTAGGG)n repeats as in Sidhom \DIFdelbegin \textit{\DIFdel{et al.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \DIFadd{et al. }\DIFaddend \citeyear{sidhom2020karyological}), (vi) number of ITS (considering only interstitial and peritelomeric signals present in both homologs, thus excluding centromeric and standard telomeric positive signals; i.e. \DIFdelbegin \DIFdel{“true” }\DIFdelend \DIFaddbegin \DIFadd{`true' }\DIFaddend ITS \textit{sensu} \citealt{bolzan2012chromosomal,chirino2017chromosomal}), and (vii) and the morphology of each macro-chromosome pair following previous classification based on the centromeric index (metacentric, submetacentric, subtelocentric and telocentric; \citealt{levan1964nomenclature}). 
We collated chromosome data for additional chameleon species from the literature. 
The full dataset along with literature references is available [details removed to anonymize submission].
\DIFdelbegin \DIFdel{Detailed results of the cytogenetic analyses are in the Supplementary Materials (Figures A12 - A17)}\DIFdelend \DIFaddbegin 

\DIFadd{We did not analyse sex chromosome evolution in chameleons for several reasons. 
Nucleotide and morphological evolution of the heterogametic sex chromosome (Y/W) is independent from autosome evolution and due to the loss of recombination the sex-specific chromosome (Y/W) is considered evolutionary isolated \mbox{%DIFAUXCMD
\citep[see e.g.][]{ezaz2014repetitive,mezzasalma2021lizards}}%DIFAUXCMD
. 
The rare cases where multiple sex chromosome systems occur due to a sex chromosome-autosome fusion concern intraspecific rearrangement \mbox{%DIFAUXCMD
\citep[as in the case of the neo-sex chromosomes in Furcifer reported by][]{rovatsos2015female,rovatsos2017evolution,rovatsos2019zz}}%DIFAUXCMD
, which would not be relevant in our inter-/intra-generic macroevolutionary analysis. 
In addition, data on chameleon sex chromosomes are currently available only for }\textit{\DIFadd{Furcifer}} \DIFadd{\mbox{%DIFAUXCMD
\citep{rovatsos2015female,rovatsos2017evolution,rovatsos2019zz} }%DIFAUXCMD
and }\textit{\DIFadd{Chameleo}} \DIFadd{\mbox{%DIFAUXCMD
\citep{sidhom2020karyological,tishakova2022identification} }%DIFAUXCMD
(1-2 species with hypothesised sex chromosomes which are morphologically indistinguishable). 
More data on the other chameleon genera should be gathered to better understand sex chromosome evolution in the family (possibly testing the homology of different sex chromosome systems), which, however, should be treated as a distinct topic}\DIFaddend .

\subsection{Analyses}
All analyses except the chromosome evolutionary models (see below) were carried out in R \citep{R}, and R code to reproduce our analyses is available at [details removed to anonymize submission] (Zenodo DOI will be added on acceptance). 
We \DIFdelbegin \DIFdel{visualised }\DIFdelend \DIFaddbegin \DIFadd{visualized }\DIFaddend karyotype data (chromosome number, arm number, number of macrochromosome pairs, and number of microchromosome pairs) on our phylogeny (Supplementary Materials: Figures \DIFdelbegin \DIFdel{A1 and A2}\DIFdelend \DIFaddbegin \DIFadd{A5 and A6}\DIFaddend ) using the ggtree R package \citep{yu2017ggtree}.

\subsubsection{Chromosome evolutionary models}
Some earlier papers have treated chromosome number as a continuous trait and modelled it using standard phylogenetic comparative methods for continuous data (e.g. \citealt{vershinina2017evolutionary}). 
However, chromosome number is more appropriately \DIFdelbegin \DIFdel{modelled }\DIFdelend \DIFaddbegin \DIFadd{modeled }\DIFaddend as a discrete trait which changes in predictable ways. 
Chromosome numbers can increase due to single chromosome gains (ascending dysploidy), chromosome fissions, whole genome duplications (polyploidy, leading to a doubling of the chromosome number), or half genome duplications (demi-polyploidy rate, leading to a 1.5 times increase in the chromosome number), and decrease due to single chromosome losses (descending dysploidy), or chromosome fusions. 
To model chromosome evolution in chameleons we therefore used ChromEvol v.2.0 \citep{glick2014chromevol,mayrose2010probabilistic} which fits likelihood models to the evolution of chromosome numbers, using a phylogeny (excluding the outgroup) and the haploid number of chromosomes (n) at each tip as input data. 
We manipulated outputs from ChromEvol using the ChromEvol R package \citep{chromevol}.

We fitted two different models of chromosome evolution; a Constant Rates model which \DIFdelbegin \DIFdel{optimises }\DIFdelend \DIFaddbegin \DIFadd{optimizes }\DIFaddend (i) the rate of gain of a single chromosome; and (ii) the rate of loss of a single chromosome; and a Linear Rates model which additionally \DIFdelbegin \DIFdel{optimises }\DIFdelend \DIFaddbegin \DIFadd{optimizes }\DIFaddend (iii) linear dependency between the current haploid number and the rate of gain chromosomes; and (iv) linear dependency between the current haploid number and the rate of loss chromosomes. 
We focus on models that do not include polyploidization or demi-polyploidization rates because these processes rarely occur in squamates and are unknown in Chamaeleonidae \citep{bogart1980evolutionary,mezzasalma2021lizards}, however, we report the results from these additional models in the supplemental for completeness (Supplementary Materials: Tables \DIFdelbegin \DIFdel{A1 - }\DIFdelend A2 \DIFaddbegin \DIFadd{and A3}\DIFaddend ). 

ChromEvol requires a phylogeny with branch lengths that represent the expected number of chromosome-number transitions along a branch. We therefore scaled the tree so that the total tree length was equal to the number of unique chromosome counts as suggested in Glick and Mayrose \DIFdelbegin \DIFdel{\mbox{%DIFAUXCMD
\citeyear{glick2014chromevol}}%DIFAUXCMD
}\DIFdelend \DIFaddbegin \DIFadd{(\mbox{%DIFAUXCMD
\citeyear{glick2014chromevol}}%DIFAUXCMD
)}\DIFaddend . 
We back-transformed all parameter estimates using this scaling parameter before reporting them (see Results). 
We ran each model 10,000 times, with the maximum and minimum number of chromosomes allowed to be 10 units larger/smaller than the maximum/minimum chromosome number observed in the data (n = 19 and n = 10 respectively). 
Finally, we used the Akaike Information Criterion (AIC) to identify the best fitting model. 

The model outputs of ChromEvol vary depending on the root frequencies, i.e. the probabilities of each number of chromosomes being the root state for the tree. 
We either (1) set root frequencies as 0.333 for n = 16, n = 17 and n = 18, because the most common chromosome numbers found in Iguania \DIFaddbegin \DIFadd{('Iguania' in Table 1)}\DIFaddend , the group within which Chamaeleonidae is nested, are 2n = \DIFdelbegin \DIFdel{32–36 }\DIFdelend \DIFaddbegin \DIFadd{32-36 }\DIFaddend \citep{olmo2005chromorep}; (2) fixed root frequencies so the root was n = 18, because the \DIFdelbegin \DIFdel{hypothesised }\DIFdelend \DIFaddbegin \DIFadd{hypothesized }\DIFaddend ancestral karyotype for lizards is 2n = 36 \citep{deakin2016anchoring,lisachov2021whole,mezzasalma2021lizards} \DIFaddbegin \DIFadd{('n $=$ 18' in Table 1)}\DIFaddend ; or (3) estimated the root frequencies using the \DIFdelbegin \DIFdel{model}\DIFdelend \DIFaddbegin \DIFadd{chromEvol model ('estimated' in Table 1)}\DIFaddend . 
We also repeated each model excluding \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend }. 
This species has an unusually high number of chromosomes for a chameleon of 2n = 62 (the next highest is 2n = 38), so likely represents an unusual instance of karyotypic evolution which may skew our model outputs. 

\DIFaddbegin \DIFadd{ChromEvol assumes all changes in chromosome numbers occur along branches of the phylogeny, i.e. anagenetically. 
However, chromosomal rearrangements may also occur at speciation events, i.e. cladogenetically. 
To incorporate anagenetic and cladogenetic chromosomal changes into our analyses we used the ChromoSSE model \mbox{%DIFAUXCMD
\citep{freyman2018cladogenetic} }%DIFAUXCMD
implemented in RevBayes \mbox{%DIFAUXCMD
\citep{10.1093/sysbio/syw021}}%DIFAUXCMD
. 
These models are computationally intensive, so rather than fitting all the models described above, we used the best fitting model identified in the chromEvol chromosome evolution analyses above (Constant Rates, removing }\textit{\DIFadd{Rieppeleon brevicaudatus}}\DIFadd{, and using n = 18 as the root node; see Results) to specify our analysis. 
We fitted three cladogenetic chromosomal change parameters (i) speciation without chromosomal changes, (ii) speciation associated with chromosome gain, and (iii) speciation associated with chromosome loss, and two anagenetic chromosomal change parameters which optimized (i) the rate of gain of a single chromosome; and (ii) the rate of loss of a single chromosome (i.e. the Constant Rates model in chromEvol). 
We used a single species turnover rate, set the maximum number of chromosomes to n = 29, and used the default ChromoSSE priors except for root frequencies which were fixed at n = 18. 
We ran the model for 10,000 iterations sampling at every 1,000 iterations and discarding the first 2,500 iterations as burn-in. 
We assessed convergence using functions within the RevGadgets R package \mbox{%DIFAUXCMD
\citep{https://doi.org/10.1111/2041-210X.13750}}%DIFAUXCMD
. 
All model parameters had a mean effective sample size (ESS) of over 200, and traceplots indicated that models had converged.
}

\subsubsection{\DIFadd{Correlations between chromosome evolutionary change and historical events}}
\DIFadd{Madagascar and East Africa have undergone significant environmental change since the Cretaceous \mbox{%DIFAUXCMD
\citep[e.g.][]{ohba2016madagascar,linder2017east,genin2022co,jiao2023linking}}%DIFAUXCMD
. 
Therefore, to examine whether any of the chromosome evolutionary changes in chameleons are associated with major geotectonic or climatic events, we evaluated the results with comparisons to key dates from the literature. 
}

\DIFadd{The dates and events we selected were (A) 33.9 Ma: global cooling near the Eocene-Oligocene boundary \mbox{%DIFAUXCMD
\citep[Grande Coupure;][]{liu2009global,samonds2013imperfect}}%DIFAUXCMD
; (B) 30.0 Ma: Oligocene: Malagasy uplift \mbox{%DIFAUXCMD
\citep{stephenson2021cenozoic} }%DIFAUXCMD
and fall in sea levels; (C) 23.0 Ma: Oligocene–Miocene cooling \mbox{%DIFAUXCMD
\citep{zachos1997orbitally}}%DIFAUXCMD
; (D-E) 14 to 18 Ma: global temperature increases at the Miocene Climatic Optimum \mbox{%DIFAUXCMD
\citep{zhou2012out}}%DIFAUXCMD
, increases in Malagasy uplift \mbox{%DIFAUXCMD
\citep{stephenson2021cenozoic,jiao2023linking}}%DIFAUXCMD
, and development of the Indian monsoons \mbox{%DIFAUXCMD
\citep{samonds2013imperfect}}%DIFAUXCMD
; and (F) 10 Ma: regional volcanism \mbox{%DIFAUXCMD
\citep{michon2016volcanism} }%DIFAUXCMD
including perhaps formation of Mayotte volcano, the oldest volcano in the Comoros, and increases in regional monsoon intensity \mbox{%DIFAUXCMD
\citep[but see][]{masquelet2022east}}%DIFAUXCMD
. 
}

\DIFadd{Several of these events have previously been hypothesised to have had a potential impact on the evolution of regional and global biota \mbox{%DIFAUXCMD
\citep[e.g.][]{zhou2012out,samonds2013imperfect,buerki2013abrupt,godfrey2020mid}}%DIFAUXCMD
.
We plotted these dates onto our phylogeny along with the ancestral state estimates for chromosome numbers taken from the ChromoSSE model (see above) and, specifically, looked for parallel changes in chromosome number among the clades related to these events.
}

\DIFaddend \subsubsection{Fissions and fusions and ITS}
Single chromosome gains are predicted to occur due to chromosome fissions, whereas chromosome losses are predicted to occur due to chromosome fusions. 
To test whether fissions and fusions were likely to have occurred in the evolution of chameleon chromosome numbers, we used \DIFdelbegin \DIFdel{generalised }\DIFdelend \DIFaddbegin \DIFadd{generalized }\DIFaddend linear models (GLMs) with Poisson errors to test for correlation between numbers of macro- and micro-chromosome pairs in each chameleon karyotype. 
A significant \DIFaddbegin \DIFadd{negative }\DIFaddend correlation suggests that fission or fusion may explain variation in numbers of chromosomes across chameleons\DIFaddbegin \DIFadd{, because if the micro-chromosomes fuse to become macro-chromosomes (and vice versa) they should be negatively correlated}\DIFaddend .
To determine whether fission or fusion was more likely, we used the parameter estimates from the best fitting chromosome evolution model (see above). 
In addition, we tested for correlations between ITS and haploid number of chromosomes, the number of macrochromosome pairs and the number of microchromosome pairs, also using GLMs but with quasipoisson errors due to overdispersion in these models.

\subsubsection{Phylogenetic patterns}
To investigate how chromosome number is related to phylogenetic relatedness, we calculated the phylogenetic distance (in millions of years) between each pair of taxa in the phylogeny, and plotted this estimate against their differences in chromosome number. 
Note that we could not use standard measures of phylogenetic signal (e.g. Pagel's $\lambda$; \citealt{pagel1999inferring}, Blomberg's K; \citealt{blomberg2003testing}) because chromosome number is a discrete trait with multiple levels, not continuous or binary as required by these methods. 

In addition, we simulated chromosome numbers on the tree with taxa as tips using ChromEvol v.2.0 \citep{glick2014chromevol,mayrose2010probabilistic}. 
We performed 1,000 simulations using the \DIFdelbegin \DIFdel{optimised }\DIFdelend \DIFaddbegin \DIFadd{optimized }\DIFaddend parameters taken from the best fitting model identified in the chromosome evolution analyses above (Constant Rates, removing \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend }, and using n = 18 as the root node, see Results). 
We then compared the distribution of simulated chromosome numbers at the tips of the tree to the observed distribution. 

\subsubsection{Relationships among chromosome numbers, ecology and life history}
We tested for correlations between haploid numbers of chromosomes (n) and the following variables; (1) maximum snout vent length (mm; n = 86); (2) substrate (arboreal, terrestrial or multiple; n = 77); (3) reproductive mode (oviparous or viviparous; n = 71); (4) minimum clutch size (eggs; n = 58); (5) maximum clutch size (eggs; n = 58); (6) minimum breeding age (months; n = 43); (7) maximum breeding age (months; n = 43); (8) biogeographic realm (Afrotropical, Madagascar, \DIFdelbegin \DIFdel{Oriental }\DIFdelend \DIFaddbegin \DIFadd{Indomalayan }\DIFaddend or Palearctic; n = 86); (9) absolute latitude (decimal degrees; DD; n = 86). 
All ecological and life history data were taken from Meiri (\citeyear{meiri2018traits}). 
Continuous variables were mean-centred and scaled to unit variance prior to analyses.

We used Bayesian phylogenetic \DIFdelbegin \DIFdel{generalised }\DIFdelend \DIFaddbegin \DIFadd{generalized }\DIFaddend mixed models (GLMMs) with Poisson errors in the R package MCMCglmm \citep{hadfield2010mcmc} to test for correlations, including the phylogeny (as the inverse of the phylogenetic variance-covariance matrix) as a random effect to account for phylogenetic autocorrelation. 
We ran each MCMCglmm model for 1 x 106 iterations sampling at every 1,000 iterations and discarding the first 1 x 105 iterations as burn-in. 
We used the default priors for MCMCglmm (\DIFdelbegin \DIFdel{$μ = 0$ }\DIFdelend \DIFaddbegin \DIFadd{$\mu = 0$ }\DIFaddend and $V = I × 1010$ for fixed effects and parameter expanded priors, and $V = 1$, $\nu = 1$, $\alpha \mu = 0$, and $\alpha V = 252$ for the phylogenetic random effects). 
All model parameters had a mean effective sample size (ESS; estimated using the R package coda; \citealt{plummer2006coda}) of over 800, and traceplots indicated that models had converged.

%-------------------------------------------------------------------------------
\section{Results}

\DIFaddbegin \subsection{\DIFadd{Cytogenetic analyses}}

\DIFadd{Standard karyotyping was successfully performed on all the considered species (n = 54) but }\textit{\DIFadd{Calumma cucullatum}} \DIFadd{and }\textit{\DIFadd{Calumma}} \DIFadd{sp. 4, which did not provide suitable metaphase plates.
Considering the quality and number of metaphase plates, Ag-NOR staining was performed on all the }\textit{\DIFadd{Brookesia}} \DIFadd{and }\textit{\DIFadd{Palleon}} \DIFadd{species here considered, 13 }\textit{\DIFadd{Calumma}} \DIFadd{species and 11 }\textit{\DIFadd{Furcifer}} \DIFadd{species. 
Similarly, TELO-FISH was successfully performed on nine }\textit{\DIFadd{Brookesia}}\DIFadd{, 13 }\textit{\DIFadd{Calumma}} \DIFadd{and 12 }\textit{\DIFadd{Furcifer}} \DIFadd{species, respectively (see Figures \ref{fig-s1} - \ref{fig-s6}).
Chromosome number was found to be constant (2n = 36, AN = 48) in all the }\textit{\DIFadd{Brookesia}} \DIFadd{species (n = 13) and in }\textit{\DIFadd{Palleon nasus}} \DIFadd{(Figure \ref{fig-s1}), and invariably with six macro- and 12 micro-chromosome pairs.
}

\noindent \DIFadd{All the macrochromosome pairs resulted metacentric with the only exception of the 2nd submetacentric pair in }\textit{\DIFadd{P. nasus}}\DIFadd{. 
No heteromorphic chromosome pairs were found in any }\textit{\DIFadd{Brookesia}} \DIFadd{species or in }\textit{\DIFadd{P. nasus}}\DIFadd{.
No ITS signals were found in the karyotype of any }\textit{\DIFadd{Brookesia}} \DIFadd{species or in }\textit{\DIFadd{P. nasus}} \DIFadd{(Figure \ref{fig-s2}).
}

\noindent \DIFadd{In the genus }\textit{\DIFadd{Calumma}}\DIFadd{, karyotypes varied from 2n = 38 (in }\textit{\DIFadd{C. tarzan}} \DIFadd{and }\textit{\DIFadd{C. amber}}\DIFadd{) to 2n = 28 (in }\textit{\DIFadd{C. nasutum}} \DIFadd{and }\textit{\DIFadd{C. fallax}} \DIFadd{clade 2) (AN = 40-56), with 2n = 36 resulting the most common chromosome configuration in the genus (in 10 out of 24 species studied; Figure \ref{fig-s3}). 
Karyotype of 2n = 34, 32 and 30 chromosomes were showed by four, three and four species, respectively. 
A variable number of macro- (6-9) and micro-chromosome pairs (7-13) was also recorded among the studied species of the genus. 
Chromosome morphology is also variable in }\textit{\DIFadd{Calumma}}\DIFadd{, macrochromosome pairs are generally biarmed (from meta- to sub-telocentric pairs), but telocentric elements were observed in }\textit{\DIFadd{Calumma}} \DIFadd{sp. 1 (7th pair). 
}

\noindent \DIFadd{A heteromorphic pair was observed in the karyotype of }\textit{\DIFadd{C. crypticum}}\DIFadd{. 
NOR bearing pairs were detected in 14 species, on the 1st (three species) or 2nd chromosome pair (11 species). 
ITS signals (n = 2-4) were found in six species showing karyotypes from 2n = 36 to 2n = 28 (Figure \ref{fig-s4}).
}

\noindent \DIFadd{The genus }\textit{\DIFadd{Furcifer}} \DIFadd{showed karyotypes ranging from 2n = 34 (only in }\textit{\DIFadd{F. balteatus}}\DIFadd{) to 2n = 22 (the most common karyotype configuration in the genus, found in eight out of 16 species studied; AN = 34-54; Figure \ref{fig-s5}). 
Karyotypes of 2n = 28 were showed by two species (}\textit{\DIFadd{F. wilsii}} \DIFadd{and }\textit{\DIFadd{F. labordi}}\DIFadd{), 2n = 26 by one species (}\textit{\DIFadd{F. campani}}\DIFadd{) and 2n = 24 by five species. 
}

\noindent \DIFadd{Macrochromosome pairs ranged from six to 13 pairs, while microchromosomes from one to 11 pairs. 
Chromosome morphology is mostly biarmed in the genus (meta- and submeta-centric pairs are common in the genus), but 1-2 telocentric pairs are present in the karyotype of }\textit{\DIFadd{F. belalandensis}} \DIFadd{and }\textit{\DIFadd{F. minor}}\DIFadd{.
Heteromorphic chromosome pairs were found in seven species, usually on the last macrochromosome pair. 
In the species analysed here, loci of NOR are always on macrochromosomes (1st, 2nd, 3rd or 4th pair). 
A highly variable number of ITS signals (n = 2-14) were found in 11 species with 2n = 34-22 (Figure \ref{fig-s6}).
}

\DIFaddend \subsection{Chromosome evolutionary models}
When \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend } was included, the Linear Rates model fitted better than the Constant Rates model (Table \ref{table-models}).
Although rates of chromosome loss were \DIFdelbegin \DIFdel{consistently }\DIFdelend \DIFaddbegin \DIFadd{generally }\DIFaddend higher than rates of chromosome gain in these models, rates of linear dependency between the current haploid number and the rate of loss of chromosomes were \DIFaddbegin \DIFadd{generally }\DIFaddend lower for loss than gain of chromosomes, regardless of root frequency. 
The Constant Rates model fitted better than the Linear Rates model when \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend } was excluded from the analyses and rates of chromosome loss were also \DIFdelbegin \DIFdel{consistently }\DIFdelend \DIFaddbegin \DIFadd{generally }\DIFaddend higher than rates of chromosome gain in these models, regardless of root frequency (Table \ref{table-models}). 
Overall, the best fitting model was the Constant Rates model where \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend } was excluded, and the root was set to n = 18 (AIC = \DIFdelbegin \DIFdel{320.5}\DIFdelend \DIFaddbegin \DIFadd{320.7}\DIFaddend ; Table \ref{table-models}\DIFdelbegin \DIFdel{; Figure \ref{fig-best})}\DIFdelend \DIFaddbegin \DIFadd{).
}

\DIFadd{In the ChromoSSE models, the 95\% credible intervals for each parameter were as follows (i) speciation without chromosomal changes: 0.042-0.086 (MAP = 0.067); (ii) speciation associated with chromosome gain: 0.0002-0.020 (MAP = 0.002); (iii) speciation associated with chromosome loss: 0.001-0.029 (MAP = 0.008); (iv) anagenetic rate of gain of a single chromosome: 0.009-0.070 (MAP = 0.029); and (v) anagenetic rate of loss of a single chromosome: 0.044-0.110 (MAP = 0.073). 
Ancestral states for chromosome numbers estimated from this model are shown in Figure \ref{fig-best}}\DIFaddend .

\DIFaddbegin \subsection{\DIFadd{Correlations between chromosome evolutionary change and historical events}}
\DIFadd{A version of Figure \ref{fig-best} with key evolutionary dates highlighted is shown in the Supplemental Materials: Figure A7.
The estimated timing of some chromosomal fusions does occasionally coincide with geotectonic and climatic events but there are many occasions where they do not. 
There are also several lineages that do not undergo changes despite the events. 
Thus, there are no clear correlations between estimated evolutionary changes in chromosome number and geotectonic and climatic events. 
The ancestors of }\textit{\DIFadd{Trioceras}} \DIFadd{and }\textit{\DIFadd{Calumma}} \DIFadd{may have acquired n = 18 chromosomes independently around the same time ($\sim$25 Ma) and ancestors of }\textit{\DIFadd{Chamaeleo}} \DIFadd{and }\textit{\DIFadd{Furcifer}} \DIFadd{may have acquired n = 12 chromosomes independently around the same time ($\sim$10 Ma). 
However, several lineages of }\textit{\DIFadd{Brookesia}} \DIFadd{and }\textit{\DIFadd{Calumma}} \DIFadd{are estimated to have retained a chromosome number of n = 18 despite the many global and regional environmental variations since 20 Ma. 
}

\DIFaddend \subsection{Fissions and fusions and ITS}
There was a significant negative correlation between the haploid number of chromosomes and the number of macrochromosome pairs (GLM: $\chi^2$ = 4.991, df = 1,91, p = 0.025; Figure \ref{fig-chroms}A), and a significant positive correlation between the haploid number of chromosomes and the number of microchromosome pairs (GLM: $\chi^2$ = 55.32, df = 1,91, p $<$ 0.001; Figure \ref{fig-chroms}B). 
Additionally, there was a significant relationship between the number of micro- and macro- chromosome pairs (GLM: $\chi^2$ = 97.15, df = 1,91, p $<$ 0.001; Figure \ref{fig-chroms}C). 

There was a significant negative correlation between ITS and the haploid number of chromosomes (GLM: F = 23.91, df = 1,42, p $<$ 0.001; Figure \ref{fig-its}A), a significant positive correlation between ITS and the number of macrochromosome pairs (GLM: F = 5.828, df = 1,42, p = 0.020; Figure \ref{fig-its}B), and a significant relationship between ITS and the number of microchromosome pairs (GLM: F = 25.98, df = 1,42, p $<$ 0.001; Figure \ref{fig-its}C). 

\subsection{Phylogenetic patterns}
Differences in chromosome number did not increase with phylogenetic distance (Figure \ref{fig-pairwise}); even some of the most distantly related taxa in our phylogeny shared the same chromosome numbers. 
Large differences in chromosome numbers, however, only occur at moderate to large phylogenetic distances. 

Simulations give a reasonable approximation of observed chromosome numbers at the tips of the phylogeny, however, model predictions do not account well for the large numbers of species with n = 11 \DIFdelbegin \DIFdel{, n = 12 }\DIFdelend and n = \DIFdelbegin \DIFdel{18 }\DIFdelend \DIFaddbegin \DIFadd{12 }\DIFaddend chromosomes (Figure \ref{fig-predict}).

\subsection{Relationships among chromosome numbers, ecology and life history}
We found no significant correlations among chromosome numbers and any of our ecology or life history variables (Supplementary Materials: Table \DIFdelbegin \DIFdel{A3; Figures A4-A5}\DIFdelend \DIFaddbegin \DIFadd{A4; Figures A8 and A9}\DIFaddend ). 

%-------------------------------------------------------------------------------
\section{Discussion}

That chromosomal rearrangements have contributed to the evolutionary changes in karyotypes is not controversial or new (e.g. \citealt{deakin2016anchoring}), but when, how, and how often they have contributed is less clear. 
In addition, chromosome rearrangements among closely-related species are usually tracked with molecular cytogenetics \citep{lisachov2021whole}, but the dynamics of chromosome evolutionary changes across vertebrate groups have rarely been tested with macroevolutionary methods. 
Our study provides strong evidence that in the diverse lineage of Chamaeleonidae, regular evolutionary fusion of microchromosomes has resulted in multiple, independent origins of karyotypes with a reduced number of chromosomes (from 2n = 36 to 2n = 24-22; Figure \ref{fig-best}). 
This intriguing result suggests that in chameleons a single dynamic of chromosome evolution was responsible for both convergent and divergent evolution of karyotypes. 
Below we discuss chromosome loss and karyotypic convergence in relation to previous studies, address some caveats of using ITSs to infer fusion events, the apparent lack of correlation between chameleon karyotypes and natural history traits, and speculate why so many microchromosomes have fused in chameleons.    

\subsection{Reduction, status, and convergence during chromosome evolution}
Using GLMs, we found a strong negative correlation between the number of macro- and micro-chromosomes (Figure \ref{fig-chroms}C). 
This result is consistent with a reduction in the number of microchromosomes with an increase in the number of macrochromosomes, as would be expected following fusion events in asymmetrical karyotypes. 
Even more compellingly, we found a positive correlation between the number of macrochromosomes and ITSs which is clearly consistent with fusion of microchromosomes into macrochromosomes in those species with reduced chromosome numbers (Figure \ref{fig-its}B). 
We also observed putative convergent reductions in several genera including \textit{Calumma}, \textit{Furcifer}, \textit{Rhampholeon}, and \textit{Triceros}. 
Our simulation analysis supports that, compared to the \DIFdelbegin \DIFdel{hypothesised }\DIFdelend \DIFaddbegin \DIFadd{hypothesized }\DIFaddend chameleon ancestral condition (2n = 36), there are more observations of these with a reduced chromosome number (2n = 24-22) than would be expected by chance (Figure \ref{fig-predict}). 
Collectively, this result is consistent with frequent microchromosome losses during the evolution of chameleons leading to convergence in the number of chromosomes. 
This is particularly evident in different genera such as \textit{Furcifer} and \textit{Trioceros}, which have karyotypes either similar to the \DIFdelbegin \DIFdel{hypothesised }\DIFdelend \DIFaddbegin \DIFadd{hypothesized }\DIFaddend chameleon ancestral condition of 2n = 36 (2n = 34 in \textit{F. balteatus} and several 2n = 36 in \textit{Trioceros}), and karyotypes with a reduced total chromosome number (2n = 24-22) in other species (Figure \ref{fig-best}). 
We use the term ``convergence'' in a relative sense related to the number of chromosomes, as karyotypes with lower chromosome numbers showed higher variability in macrochromosome morphology (Supplementary Materials: Figure \DIFdelbegin \DIFdel{A11}\DIFdelend \DIFaddbegin \DIFadd{A10}\DIFaddend ), suggesting the occurrence of different intrachromosomal rearrangements and/or fusion events between microchromosomes. 
Thus, convergent chameleon chromosome loss occurred via unique pathways of microchromosome fusion in different lineages, a well-established phenomenon of telomeric fusion in higher eukaryotes (e.g. \citealt{heacock2004molecular}).  
Chromosome size convergence is often discussed, particularly in species with differentiated, heteromorphic sex chromosomes \citep{montiel2017discovery,kratochvil2021sex}. 
However, convergence in chromosome number is a far less frequently reported phenomenon (particularly of fusion-based convergence; see Rens \DIFdelbegin \textit{\DIFdel{et al.}} %DIFAUXCMD
\DIFdelend \DIFaddbegin \DIFadd{et al. }\DIFaddend \citeyear{rens2003reversal} for fission-based example), making chameleons a valuable study system for chromosome evolution. 
Interestingly, our results are consistent with the karyotypic orthoselection model proposed by White (\citeyear{white1973,white1975chromosome}). 
Following White's model, chromosome changes found in any given group are not random, but \DIFdelbegin \DIFdel{characterised }\DIFdelend \DIFaddbegin \DIFadd{characterized }\DIFaddend by the accumulation of similar chromosome rearrangements, eventually leading to convergent karyotype structures \citep{white1973,white1975chromosome}. 
White also suggested that different processes such as environmental selection or intrinsic chromosomal properties might explain orthoselection of karyotypes \citep{white1973,white1975chromosome}.

Despite the evidence that chromosome fusion regularly occurred during the evolution of chameleons, this tendency was not true of all lineages we studied. Assuming an ancestral karyotypes of 2n = 36 \DIFdelbegin \DIFdel{\mbox{%DIFAUXCMD
\citep{rovatsos2017evolution} }%DIFAUXCMD
(see Materials and Methods)}\DIFdelend \DIFaddbegin \DIFadd{\mbox{%DIFAUXCMD
\citep[][see Materials and Methods]{rovatsos2017evolution}}%DIFAUXCMD
}\DIFaddend , we observed several chameleon genera with ancestral conservation of karyotypes (chromosomal evolutionary stasis; \textit{Brookesia}, \textit{Bradypodion}, \textit{Kinyongia}, \textit{Chamaeleo}; Figure \ref{fig-best}), highlighting that divergent evolutionary trends such as karyotypic orthoselection and chromosomal evolutionary stasis likely \DIFdelbegin \DIFdel{characterise }\DIFdelend \DIFaddbegin \DIFadd{characterize }\DIFaddend different evolutionary lineages. 
Furthermore, chromosomes in these groups also had a more conserved morphology than the genera with putative karyotypic reduction consistent with stasis (Supplementary Materials: Figure \DIFdelbegin \DIFdel{A11}\DIFdelend \DIFaddbegin \DIFadd{A10}\DIFaddend ). 
These genera with conserved karyotypes suggest that the tendency for microchromosomes to fuse may either (i) not always be advantageous or (ii) be a karyotypic trait that is lost (or gained). 

\subsection{Alternative explanations for observed patterns of chromosome fusions}
In addition to the fusion of chromosomes, the presence of ITSs can also be explained by chromosome inversions \citep{bolzan2017interstitial}. 
However, we think this possibility is unlikely to change the interpretation that many of the ITSs we observed are explained by fusion events. 
Specifically, if the ITSs we observed were mostly the result of chromosome inversions, we should not have observed the positive correlation between number of macrochromosomes and number of microchromosomes (Figure \ref{fig-chroms}C). 
We \DIFdelbegin \DIFdel{recognise }\DIFdelend \DIFaddbegin \DIFadd{recognize }\DIFaddend that the relationship between number of ITSs and number of macrochromosomes, while significant, is not completely linear (Figure \ref{fig-its}B), which suggests other processes (such as chromosome inversions and/or the amplification of DNA repeats) are likely involved in generating some of the ITSs we observed.
\DIFaddbegin \DIFadd{Furthermore, in some cases ITSs may not be detectable after fusions. 
This is likely due to the effect of purifying selection targeting non-essential DNA repeats and resulting in their deletion or to nucleotide substitutions producing degenerate sequences which are no more detectable using the same molecular probes \mbox{%DIFAUXCMD
\citep[see e.g.][]{lin2008endings,aksenova2019beginning}}%DIFAUXCMD
.
}\DIFaddend 

\subsection{What about \DIFdelbegin \textit{\DIFdel{Rieppeleon kerstenii}}%DIFAUXCMD
\DIFdelend \DIFaddbegin \textit{\DIFadd{Rieppeleon brevicaudatus}}\DIFaddend ?}
Our dataset supports chromosome loss via chromosome fusions and reduction of the total chromosome number as having occurred independently in multiple genera of chameleons. 
Most genera with higher numbers of chromosomes are matched to the putative ancestral karyotype (2n = 36), with one glaring exception; the species \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend } which has a karyotype of 2n = 62. 
It is also worth noting that two additional species (\textit{Calumma amber} and \textit{Calumma tarzan}) in our dataset have a larger-than-ancestral karyotype of 2n = 38. 
This is not surprising considering that our chromosome analyses and macroevolutionary models did not reject the occurrence of chromosome fissions in chameleon karyotypes, which likely generated a higher number of chromosomes in these taxa.

\subsection{Why \DIFdelbegin \DIFdel{has }\DIFdelend \DIFaddbegin \DIFadd{have }\DIFaddend microchromosome \DIFdelbegin \DIFdel{fusion }\DIFdelend \DIFaddbegin \DIFadd{fusions }\DIFaddend happened so frequently in chameleons?}
Microchromosome fusion is widespread in vertebrate animals \citep{waters2021microchromosomes}. 
If the reduction of the total chromosome number via fusion is so common, is that because it is beneficial? In other words, what potential benefits are there to having a symmetrical (versus asymmetrical) karyotype? 
One clear benefit would be to reinforce reproductive isolation. 
Chromosome fusions may select against introgression between diverging populations, because the potential for viable offspring is decreased between individuals with divergent karyotypes \citep{cicconardi2021chromosome}.
This incompatibility might serve to initiate or hasten speciation through rapid evolution of key traits in reproductively-isolated populations. If this hypothesis were the case in chameleons, we might expect to observe correlates between natural history traits and karyotypes, which we did not (Supplementary Materials Table \DIFdelbegin \DIFdel{A3; Figures A4-A5}\DIFdelend \DIFaddbegin \DIFadd{A4; Figures A8-A9}\DIFaddend ). 
However, it is also possible that we did not measure the relevant traits associated with speciation in chameleons, and/or that these traits are genus or population specific.
Perhaps larger chromosomes achieved through fusions may be beneficial in terms of genome structure and function? Evidence for this possibility is mixed. In some insects, there is a positive correlation between chromosome size and the functional gene conservation \citep{cicconardi2021chromosome}. 
However, in other groups like birds (and importantly some non-avian reptiles) microchromosomes tend to be higher in gene content \citep{waters2021microchromosomes}.
The types of microchromosome fusions observed in chameleons have also occurred in other squamates \citep{deakin2016anchoring}; including in lacertids (\textit{Lacerta agilis}; \citealt{srikulnath2014identification}) and notably geckos where repeated fusions have ostensibly eliminated microchromosomes altogether (\textit{Gekko \DIFdelbegin \DIFdel{hokuensis}\DIFdelend \DIFaddbegin \DIFadd{hokouensis}\DIFaddend }; \citealt{srikulnath2015karyotype}). 
Thus, a broader phylogenetic sampling of squamates with an evolutionary history of chromosome fusion would provide valuable insights for future research. 
The current lack of any obvious correlation between the reduction of the total chromosome number and life history traits \DIFdelbegin \DIFdel{in }\DIFdelend seems to support White's (\citeyear{white1973,white1975chromosome}) view on intrinsic chromosome/genome properties as the most robust explanation for orthoselection of chameleon karyotypes.

\subsection{Conclusions}
Collectively, our results suggest that the fusion of microchromosomes into macrochromosomes has been a common occurrence during the evolution of true chameleons. 
While this phenomenon has resulted in convergent evolution of chromosome number in several genera and species, it is not found to be related to the ecological, physiological, or biogeographical variables here considered. 
This result could indicate that variability in chromosome number, morphology, and patterns of convergent evolution through chromosome loss (i.e. orthoselection; \citealt{white1973}) is mostly due to intrinsic properties of the genome, as initially \DIFdelbegin \DIFdel{hypothesised }\DIFdelend \DIFaddbegin \DIFadd{hypothesized }\DIFaddend by White (\citeyear{white1975chromosome}). 
As such, our study provides additional support that the genomic predisposition of ancestors may be the most substantive predictor of chromosome change; a key driver of biological and genomic diversification.

%-------------------------------------------------------------------------------
\section{Ethics}
\DIFdelbegin \DIFdel{For this study we }\DIFdelend \DIFaddbegin \DIFadd{We }\DIFaddend used samples already collected for other research with the approval of institutional committees \DIFaddbegin \DIFadd{(see Acknowledgements). No live animals were used during this study}\DIFaddend .
\DIFdelbegin \DIFdel{No further sampling was performed
}\DIFdelend 

\DIFdelbegin \section{\DIFdel{Conflict of interest declaration}}
%DIFAUXCMD
\addtocounter{section}{-1}%DIFAUXCMD
\DIFdel{We declare we have no competing interests.
}%DIFDELCMD < 

%DIFDELCMD < %%%
\DIFdelend %-------------------------------------------------------------------------------
% References
\bibliographystyle{evolution}
\bibliography{chameleons}

%-------------------------------------------------------------------------------
\DIFaddbegin \newpage
\begin{landscape}
\DIFaddend \section{Figures and Tables}


\input{figures/table-results}
\DIFaddbegin \end{landscape}
\DIFaddend 

%DIF <  figure 1 
%DIF >  figures are large so give them a page each
\newpage
%DIF >  figure marcello s1
\begin{figure}[h]
 \centering
  \DIFdelbeginFL %DIFDELCMD < \includegraphics[width = \linewidth]{figures/new-tree-plot4.png}
%DIFDELCMD <   %%%
\DIFdelendFL \DIFaddbeginFL \includegraphics[width = \linewidth]{figures/marcello-s1.jpg}
  \DIFaddendFL \caption{\DIFdelbeginFL \DIFdelFL{Best fitting model, removing }\textit{\DIFdelFL{Rieppeleon kerstenii}} %DIFAUXCMD
\DIFdelendFL \DIFaddbeginFL \DIFaddFL{Karyotypes of }\textit{\DIFaddFL{Brookesia}} \DIFaddendFL and \DIFdelbeginFL \DIFdelFL{using n = 18 as the root frequency}\DIFdelendFL \DIFaddbeginFL \textit{\DIFaddFL{Palleon}} \DIFaddFL{stained with Giemsa}\DIFaddendFL . \DIFdelbeginFL \DIFdelFL{Numbers at }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{Squares highlight }\DIFaddendFL the \DIFdelbeginFL \DIFdelFL{nodes are the most frequent values obtained from 1,000 simulations; numbers at the tips are the observed values. Colours represent the haploid number of chromosomes at the tips, or the proportion of simulations }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{NOR-bearing chromosome pairs stained }\DIFaddendFL with \DIFdelbeginFL \DIFdelFL{each number of chromosomes as pie charts at the nodes}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{Giemsa (left) and Ag-NOR (right)}\DIFaddendFL .
}
  \DIFaddbeginFL \label{fig-s1}
\end{figure}

\newpage
%DIF >  marcello figure s2
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/marcello-s2.jpg}
  \caption{\DIFaddFL{Karyotypes of }\textit{\DIFaddFL{Brookesia}} \DIFaddFL{and }\textit{\DIFaddFL{Palleon}} \DIFaddFL{stained with TELO-FISH.
}}
  \label{fig-s2}
\end{figure}

\newpage
%DIF >  marcello figure s3
\begin{figure}[h]
 \centering
  \includegraphics[width = 0.7\linewidth]{figures/marcello-s3.jpg}
  \caption{\DIFaddFL{Karyotypes of }\textit{\DIFaddFL{Calumma}} \DIFaddFL{stained with Giemsa. Squares highlight the NOR-bearing chromosome pairs stained with Giemsa (left) and Ag-NOR (right).
}}
  \label{fig-s3}
\end{figure}

\newpage
%DIF >  marcello fig-s4
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/marcello-s4.jpg}
  \caption{\DIFaddFL{Karyotypes of }\textit{\DIFaddFL{Calumma}} \DIFaddFL{stained with TELO-FISH.
}}
  \label{fig-s4}
\end{figure}

\newpage
%DIF >  marcello figure s5
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/marcello-s5.jpg}
  \caption{\DIFaddFL{Karyotypes of }\textit{\DIFaddFL{Furcifer}} \DIFaddFL{stained with Giemsa. Squares highlight the NOR-bearing chromosome pairs stained with Giemsa (left) and Ag-NOR (right).
}}
  \label{fig-s5}
\end{figure}

\newpage
%DIF >  marcello fig-s6
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/marcello-s6.jpg}
  \caption{\DIFaddFL{Karyotypes of }\textit{\DIFaddFL{Furcifer}} \DIFaddFL{stained with TELO-FISH.
}}
  \label{fig-s6}
\end{figure}

%DIF >  figure 7
\newpage
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/ChromoSSE_plot.png}
  \caption{\DIFaddFL{Ancestral estimates of chromosome numbers from a ChromoSSE model, removing }\textit{\DIFaddFL{Rieppeleon brevicaudatus}} \DIFaddFL{and using n = 18 as the root frequency. Numbers at the nodes are the states with the highest posterior probability in the ChromoSSE model; numbers at the tips are the observed values. Colours represent the haploid number of chromosomes, and the size of the points at the nodes represents the posterior probability.
}}
  \DIFaddendFL \label{fig-best}
\end{figure}

%DIF <  figure 2
%DIF >  figure 8
\newpage
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/micro-macro-chromosomes.png}
  \caption{Correlations \DIFdelbeginFL \DIFdelFL{among }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{between (A) }\DIFaddendFL the haploid number of chromosomes (n) \DIFdelbeginFL \DIFdelFL{, }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{and }\DIFaddendFL numbers of macrochromosome pairs\DIFaddbeginFL \DIFaddFL{; (B) the haploid number of chromosomes (n) }\DIFaddendFL and numbers of microchromosome pairs\DIFdelbeginFL \DIFdelFL{in chameleons}\DIFdelendFL \DIFaddbeginFL \DIFaddFL{; and (C) numbers of macrochromosome pairs and numbers of microchromosome pairs}\DIFaddendFL . Fitted lines and standard errors are the outputs from \DIFdelbeginFL \DIFdelFL{generalised }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{generald }\DIFaddendFL linear models with Poisson errors. The outlier at n = 31 is \DIFdelbeginFL \textit{\DIFdelFL{Rieppeleon kerstenii}}%DIFAUXCMD
\DIFdelendFL \DIFaddbeginFL \textit{\DIFaddFL{Rieppeleon brevicaudatus}}\DIFaddendFL .
}
  \label{fig-chroms}
\end{figure}

%DIF <  figure 3
%DIF >  figure 9
\newpage
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/ITS-chromosomes.png}
  \caption{Correlations \DIFdelbeginFL \DIFdelFL{among }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{between (A) }\DIFaddendFL interstitial telomeric sequences (ITS) and the haploid number of chromosomes (n)\DIFdelbeginFL \DIFdelFL{, }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{; (B) ITS and }\DIFaddendFL numbers of macrochromosome pairs\DIFaddbeginFL \DIFaddFL{; }\DIFaddendFL and \DIFaddbeginFL \DIFaddFL{(C) ITS and }\DIFaddendFL numbers of microchromosome pairs\DIFdelbeginFL \DIFdelFL{in chameleons}\DIFdelendFL . Fitted lines and standard errors are the outputs from \DIFdelbeginFL \DIFdelFL{generalised }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{generalized }\DIFaddendFL linear models with quasipoisson errors. Note that we only have ITS data for 44 species.
}
  \label{fig-its}
\end{figure}

%DIF <  figure 4
%DIF >  figure 10
\newpage
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/species-pairs-distances.png}
  \caption{Phylogenetic distance (in millions of years) in relation to differences in chromosome numbers (2n) for each pair of taxa in the chameleon tree. Note that the cluster of values with chromosome differences greater than 20 are comparisons of various taxa with \DIFdelbeginFL \textit{\DIFdelFL{Rieppeleon kerstenii}} %DIFAUXCMD
\DIFdelendFL \DIFaddbeginFL \textit{\DIFaddFL{Rieppeleon brevicaudatus}} \DIFaddendFL (2n = 62).}
  \label{fig-pairwise}
\end{figure}

%DIF <  figure 5
%DIF >  figure 11
\newpage
\begin{figure}[h]
 \centering
  \includegraphics[width = \linewidth]{figures/chromevol-simulations-numbers-best.png}
  \caption{Distribution of observed (grey) and predicted (blue) haploid numbers of chromosomes in chameleons, excluding \DIFdelbeginFL \textit{\DIFdelFL{Rieppeleon kerstenii}}%DIFAUXCMD
\DIFdelendFL \DIFaddbeginFL \textit{\DIFaddFL{Rieppeleon brevicaudatus}}\DIFaddendFL . Predicted values are based on 1,000 simulations using the \DIFdelbeginFL \DIFdelFL{optimised }\DIFdelendFL \DIFaddbeginFL \DIFaddFL{optimized }\DIFaddendFL parameters taken from the best fitting model identified in the chromosome evolution analyses above (Constant Rates, removing \DIFdelbeginFL \textit{\DIFdelFL{Rieppeleon kerstenii}}%DIFAUXCMD
\DIFdelendFL \DIFaddbeginFL \textit{\DIFaddFL{Rieppeleon brevicaudatus}}\DIFaddendFL , and using n = 18 as the root frequency). Observed values were multiplied by 1,000 to aid comparisons.
}
  \label{fig-predict}
\end{figure} 

%-------------------------------------------------------------------------------
\newpage
\section{Table and figure legends}

Table 1: Results from Constant Rates and Linear Rates chromosome evolution models. root = Root frequency (see text). AIC = Akaike Information Criterion. AIC values for the best fitting model in each model set are in bold. loss = rate of chromosome loss; gain = rate of chromosome gain; lossL = linear dependency between the current haploid number and the rate of loss chromosomes; gainL = linear dependency between the current haploid number and the rate of gain chromosomes.

Figure 1: \DIFdelbegin \DIFdel{Best fitting }\DIFdelend \DIFaddbegin \DIFadd{Karyotypes of }\textit{\DIFadd{Brookesia}} \DIFadd{and }\textit{\DIFadd{Palleon}} \DIFadd{stained with Giemsa. Squares highlight the NOR-bearing chromosome pairs stained with Giemsa (left) and Ag-NOR (right).
}

\DIFadd{Figure 2: Karyotypes of }\textit{\DIFadd{Brookesia}} \DIFadd{and }\textit{\DIFadd{Palleon}} \DIFadd{stained with TELO-FISH.
}

\DIFadd{Figure 3: Karyotypes of }\textit{\DIFadd{Calumma}} \DIFadd{stained with Giemsa. Squares highlight the NOR-bearing chromosome pairs stained with Giemsa (left) and Ag-NOR (right).
}

\DIFadd{Figure 4: Karyotypes of }\textit{\DIFadd{Calumma}} \DIFadd{stained with TELO-FISH.
}

\DIFadd{Figure 5: Karyotypes of }\textit{\DIFadd{Furcifer}} \DIFadd{stained with Giemsa. Squares highlight the NOR-bearing chromosome pairs stained with Giemsa (left) and Ag-NOR (right).
}

\DIFadd{Figure 6: Karyotypes of }\textit{\DIFadd{Furcifer}} \DIFadd{stained with TELO-FISH.
}

\DIFadd{Figure 7: Ancestral estimates of chromosome numbers from a ChromoSSE }\DIFaddend model, removing \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend } and using n = 18 as the root frequency. Numbers at the nodes are the \DIFdelbegin \DIFdel{most frequent values obtained from 1,000 simulations}\DIFdelend \DIFaddbegin \DIFadd{states with the highest posterior probability in the ChromoSSE model}\DIFaddend ; numbers at the tips are the observed values. Colours represent the haploid number of chromosomes\DIFdelbegin \DIFdel{at the tips, or the proportion of simulations with each number of chromosomes as pie charts }\DIFdelend \DIFaddbegin \DIFadd{, and the size of the points }\DIFaddend at the nodes \DIFaddbegin \DIFadd{represents the posterior probability}\DIFaddend .

Figure \DIFdelbegin \DIFdel{2: Correlations among }\DIFdelend \DIFaddbegin \DIFadd{8: Correlations between (A) }\DIFaddend the haploid number of chromosomes (n) \DIFdelbegin \DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{and }\DIFaddend numbers of macrochromosome pairs\DIFaddbegin \DIFadd{; (B) the haploid number of chromosomes (n) }\DIFaddend and numbers of microchromosome pairs\DIFdelbegin \DIFdel{in chameleons}\DIFdelend \DIFaddbegin \DIFadd{; and (C) numbers of macrochromosome pairs and numbers of microchromosome pairs}\DIFaddend . Fitted lines and standard errors are the outputs from \DIFdelbegin \DIFdel{generalised }\DIFdelend \DIFaddbegin \DIFadd{generalized }\DIFaddend linear models with Poisson errors. The outlier at n = 31 is \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend }.

Figure \DIFdelbegin \DIFdel{3: Correlations among }\DIFdelend \DIFaddbegin \DIFadd{9: Correlations between (A) }\DIFaddend interstitial telomeric sequences (ITS) and the haploid number of chromosomes (n)\DIFdelbegin \DIFdel{, }\DIFdelend \DIFaddbegin \DIFadd{; (B) ITS and }\DIFaddend numbers of macrochromosome pairs\DIFdelbegin \DIFdel{and }\DIFdelend \DIFaddbegin \DIFadd{; and (C) ITS and }\DIFaddend numbers of microchromosome pairs\DIFdelbegin \DIFdel{in chameleons}\DIFdelend . Fitted lines and standard errors are the outputs from \DIFdelbegin \DIFdel{generalised }\DIFdelend \DIFaddbegin \DIFadd{generalized }\DIFaddend linear models with quasipoisson errors. Note that we only have ITS data for 44 species. 

Figure \DIFdelbegin \DIFdel{4}\DIFdelend \DIFaddbegin \DIFadd{10}\DIFaddend : Phylogenetic distance (in millions of years) in relation to differences in chromosome numbers (2n) for each pair of taxa in the chameleon tree. Note that the cluster of values with chromosome differences greater than 20 are comparisons of various taxa with \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend } (2n = 62).

Figure \DIFdelbegin \DIFdel{5}\DIFdelend \DIFaddbegin \DIFadd{11}\DIFaddend : Distribution of observed (grey) and predicted (blue) haploid numbers of chromosomes in chameleons, excluding \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend }. Predicted values are based on 1,000 simulations using the \DIFdelbegin \DIFdel{optimised }\DIFdelend \DIFaddbegin \DIFadd{optimized }\DIFaddend parameters taken from the best fitting model identified in the chromosome evolution analyses above (Constant Rates, removing \textit{Rieppeleon \DIFdelbegin \DIFdel{kerstenii}\DIFdelend \DIFaddbegin \DIFadd{brevicaudatus}\DIFaddend }, and using n = 18 as the root frequency). Observed values were multiplied by 1,000 to aid comparisons.

\end{document}